### ISSUE TITLE
FAQ issue: "How do I install the NVIDIA driver?"

### ISSUE THREAD

**Issue by blthayer** [NVIDIA/nvidia-docker#964]
Hello,

The FAQ for installing an NVIDIA driver, found [here](https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#how-do-i-install-the-nvidia-driver), has a hyperlink with the text `package manager`. Note that the "Quickstart" links to this "How do I install the NVIDIA driver" section in the FAQ.

This `package manager` link takes you [here](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#package-manager-installation), which is for the installation of CUDA. However, as is stated in the [Tensorflow documentation](https://www.tensorflow.org/install/docker) and in the [nvidia-docker README](https://github.com/NVIDIA/nvidia-docker/blob/master/README.md), the NVIDIA driver is required, but **not** the CUDA toolkit.

I spent quite a bit of time installing the CUDA toolkit and being confused as to why I needed it when it's stated I don't. I think the links for installing NVIDIA drivers should instead take you to the [NVIDIA driver download page](https://www.nvidia.com/Download/index.aspx?lang=en-us).

Alternatively, on Ubuntu I've found the [graphics-drivers ppa](https://launchpad.net/~graphics-drivers/+archive/ubuntu/ppa) to be useful in keeping my drivers up to date.

Thanks,

Brandon

**Comment by RenaudWasTaken** [comment]
Thanks for the feedback and we will be posting some information on our official documentation site on https://docs.nvidia.com/datacenter

**Comment by Drodman93** [comment]
So do you need to download CUDA toolkit as well?

**Comment by blthayer** [comment]
If you're using nvidia-docker, you do not need to download the CUDA toolkit. However, you do need the NVIDIA drivers installed. The issue is that the nvidia-docker documentation links you to CUDA directions instead of the driver downloads.

So, all I had to do was install the proper NVIDIA driver, then install nvidia-docker (I already had a working docker installation), and then I was able to get rolling using the Tensorflow Docker images to do my work.
